src/components/application_manager/rpc_plugins/rc_rpc_plugin/include/rc_rpc_plugin/resource_allocation_manager_impl.h

```diff
@@ -78,8 +78,7 @@ class ResourceAllocationManagerImpl : public ResourceAllocationManager {
       const uint32_t app_id);
 
   smart_objects::SmartObjectSPtr CreateOnRCStatusNotificationToHmi(
-      const uint32_t app_id);
-
+      const uint32_t hmi_app_id, const uint32_t mobile_app_id);
```

Comment:
@LitvinenkoIra there is no need to specify both mobile and hmi app_id. Both this app id's should belong to the same application. You leave a lot of space for error in usage of this functions ( mistmach of mobile and hmi_app id). Would be better pass app id or application, and get hmi_and mobile app-id inside.